resizer/typings.d.ts

```diff
@@ -0,0 +1,963 @@
+// Figma Plugin API version 1, update 14
```

Comment:
In other samples, these typings files are named `figma.d.ts`. Also worth checking with Anthony on this to see if he's changing it.